kokoro/windows-msvc-2017-release-bazel/build.bat

```diff
@@ -0,0 +1,73 @@
+:: Copyright (c) 2019 Google LLC.
```

Comment:
Might need to rename the file if you end up using MSVC 2015.

Reply:
Done